src/Pages/DashboardPage/Dashboard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Dashboard.css';
+import Barchart from './Chart.js';
+import Data from './data.json';
+
+const Dashboard = () => {
+    return(
+        <div className = "dashboardcontainer">
+            <div className = "chart">
+                <Barchart />
+            </div>
+            <div className = "details">
+                {Data.map( (postDetail,index) => {
```

Comment:
Data will not be an array because we will be displaying details of one particular address in Dashboard page.